src/api/services/usersService.ts

```diff
@@ -87,9 +90,39 @@ export const enrichUserWithIsApprover = async (user: User) => {
     isApprover: userIsApprover,
   };
 };
-export const getAllUserFromParticipant = async (participant: Participant) => {
+
+const usersWithCurrentParticipantUserRoleNames = async (users: User[], participantId: number) => {
```

Comment:
Is there a simpler function name for this? Its a little confusing

Reply:
renamed to `mapUsersWithParticipantRoles`